### DISCUSSION TITLE
Force line break inside shortcode block

### DISCUSSION THREAD

**Discussion by rkranendonk** [google/docsy#1696] [Q&A] [answered]
I'm using blocks/cover with a rather long Title, and the line now breaks unnaturally. Is it possible to force a break at the position I prefer?

**Comment by fekete-robert** [comment]
Have you tried adding <br/>?

**Reply by rkranendonk** [reply]
Yes, and `{{%<br/>%}}` and `{{<br/>}}`. All are rendered literally.

**Comment by fekete-robert** [comment, marked as answer]
It seems that the title attribute is used as a string. Try adding a title
explicitly within the shortcode instead, something like this:
```
{{< blocks/cover image_anchor="top" width="min" color="white" >}}
<h1 class="display-1 mt-0 mt-md-5
pb-4">slkdjlsjdgljsldkgglksndlkgnlkzdnf;kv<br/>has;dlh;lskdhg;lshdglhslkdghlkshdlkhsdglkh</h1>
```

**Reply by rkranendonk** [reply]
Thanks! Also helps me understand better how to manipulate the look-and-feel. I don't have HTML/CSS experience so this is all quite new to me.
